wasp/src/main/java/com/orhanobut/wasp/VolleyImageHandler.java

```diff
@@ -119,6 +120,7 @@ public void run() {
             Bitmap bitmap = response.getBitmap();
             if (bitmap != null) {
                 imageView.setImageBitmap(bitmap);
+                waspImage.logWaspImageResponseSuccess(bitmap);
```

Comment:
I think we can make the method name "logSuccess" , since the object is wasp image.